### ISSUE TITLE
Reason for Documenter being a dependency

### ISSUE THREAD

**Issue by theogf** [JuliaIO/GoogleDrive.jl#19]
It looks like `Documenter` is a dependency of `GoogleDrive` but is not actually used? Could it be removed?

**Comment by JeffFessler** [comment]
Hopefully fixed by #20
Since you submitted this issue, presumably you are actually trying to use the package.
Could you test your use case on the master branch and then let me know if it works for you?
If it does, I'll tag the current release despite the Docs not building properly...

**Comment by theogf** [comment]
Yeah, it is more of a dependency issue actually, I do not directly depend on GoogleDrive but `Embeddings.jl` does. I could have a look if the breaking changes create an issue

**Comment by theogf** [comment]
Actually looking at the code, I could not find anything breaking that would lead to 0.2 ?

**Comment by JeffFessler** [comment]
Ah good point, we just added functions, so I guess 0.1.2 is the appropriate bump.  I'll fix that.

**Comment by JeffFessler** [comment]
ok, working on tagging the release but there is a hiccup due to the repo relocation:
https://github.com/JuliaIO/GoogleDrive.jl/commit/71829fd9c53cde596e06cb3344a8376461f64a6a

**Comment by theogf** [comment]
Once https://github.com/JuliaRegistries/General/pull/92753 is merged, one should be able to make a release with 0.1.3
